tests/test_insert.py

```
@@ -40,13 +40,40 @@ def _preflight_ok():
     return ChangeInfo(current_version=10, last_read_version=10)
 
 
+def _tabs_doc(*pairs):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_tabs_doc`

(ANN202)